### PR TITLE
add setup.py file for tybalt

### PR DESCRIPTION
**Pull request by gwaygenomics**
Adding `setup.py` file to enable other repositories to use methods in the `tybalt` folder. Inspired by greenelab/hetmech#89

### CODE REVIEW

**Comment by dhimmel** [comment]
Okay just so I'm clear this repo contains:

1. the tybalt python package
2. other stuff that's not the package, i.e. a specific analysis

And the desire is to be able to use the tybalt codebase in a more formal way in the other code files here or elsewhere?

**Comment by gwaygenomics** [comment]
correct

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+setup(name='tybalt',
+      description='train autoencoders with gene expression data',
+      long_description='compress gene expression data with unsupervised learning to extract biological patterns',
+      author='Gregory Way',
+      author_email='gregory.way@gmail.com',
+      url='https://github.com/greenelab/tybalt',
+      packages=['tybalt'],
+      license='BSD 3-Clause License',
+      install_requires=['keras', 'tensorflow', 'pandas', 'scikit-learn'])
+
```
Comment (dhimmel) [inline setup.py]: Is there a min python version. You may want to specify that like:
```python
    # Specify python version
    python_requires='>=3.4',
```

**Comment by gwaygenomics** [comment]
awesome, thanks - I will likely need to reference this in the future

**Pull request merged**